.luacheckrc

```diff
@@ -10,9 +9,21 @@ read_globals = {
 	"PseudoRandom", "ItemStack",
 	"AreaStore",
 	"default",
+	"factions",
 	table = { fields = { "copy", "getn" } }
 }
 
 globals = {
-	"minetest"
+	"minetest",
+	-- mod namespace
+	"areas",
+	-- legacy
+	"IsPlayerNodeOwner",
+	"GetNodeOwnerName",
+	"HasOwner",
+	"owner_defs"
```

Comment:
How can these be acceptable globals? I'd prefer that luacheck reports those as coding errors.

Reply:
legacy globals... well, i would like to remove them but i guess they are in use _somewhere_

Reply:
These likely have their origins in the `node_ownership` mod and should be removed at least since 6 years.